extensions/amp-story/1.0/amp-story.css

```diff
@@ -321,6 +321,12 @@ amp-story-grid-layer > * {
   pointer-events: auto !important;
 }
 
+/** Ensure clicks inside anchor tags are handled by the tooltip. */
+amp-story-grid-layer a * {
+  pointer-events: none;
+  cursor: default;
```

Comment:
Since these don't have pointer events, does the cursor property actually do anything?